_meta/ecs-migration.yml

```diff
@@ -185,30 +179,25 @@
   to: context.db.content
   index: false
 
-#  # not in ECS
-#- from: context.db.user
-#  to: context.db.user
-#  index: false
-#
-#- from: context.user.email
-#  to: user.email
+  # not in ECS
+- from: context.db.user
+  to: context.db.user
```

Comment:
Why is this kept under `context` while everything else is moved outside?

Reply:
If it stays there, just remove the entry. This file is for what's being migrated :-)